docs/sample_config.yaml

```diff
@@ -33,10 +33,16 @@ server_name: "SERVERNAME"
 #
 pid_file: DATADIR/homeserver.pid
 
-# The path to the web client which will be served at /_matrix/client/
-# if 'webclient' is configured under the 'listeners' configuration.
+# Absolute URL to the web client to be redirected from /_matrix/client/
+# (and also root URL) if 'webclient' is configured under the 'listeners'
+# configuration.
```

Comment:
Minor wording change:

> The absolute URL to the web client which /_matrix/client will redirect to if 'webclient' is configured under the 'listeners' configuration.

I'm not sure if it makes sense to include the note about the root URL, but don't feel strongly.

(Also update `synapse/config/server.py` with the same changes please!)